src/backend/metal/src/lib.rs

```diff
@@ -143,6 +143,7 @@ impl Instance {
         window::Surface {
             inner: Arc::new(self.create_from_nsview(nsview)),
             apply_pixel_scale: false,
+            has_swapchain: false
```

Comment:
ok, now I realize the ugliness of the hack: in Metal we can't really create multiple swapchains, since each one will think it's a sole owner of surface's drawables. Unless.... we somehow make a `CAMetalLayer` per swapchain.